src/bson.c

```diff
@@ -767,14 +796,66 @@ static const bson_visitor_t php_bson_visitors = {
 	{ NULL }
 };
 
+static bool map_element_matches_field_path(php_phongo_field_path_map_element* map_element, php_phongo_field_path* current)
```

Comment:
I realize these are static, but shouldn't we be prefixing with `php_phongo` anyway for good measure?

Reply:
This is the one that I was pondering about a little, I think I would lean to "no". It makes it much clearer which ones are statics, and we also benefit from a shorter name, and much easier auto-completion in the editors.